.github/labeler.yml

```diff
@@ -32,6 +32,7 @@ BUILD:
   - "**/*gradle*"
 DOCS:
   - docs/**/*
+  - site/**/*
```

Comment:
Can we fix this in a separate PR? It isn't necessary for this one and it would be great to get it in faster.